src/main/java/Data/BookHandler.java

```diff
@@ -29,7 +29,6 @@ public List<Book> readAllBooks() { // this method reads the book data from a fil
         } catch (IOException exception) {
             exception.printStackTrace();
```

Comment:
Better another exception handling. PrintStacktrace is not user friendly.

Reply:
New exception class created: 

package repositories;

import java.io.IOException;

public class BookSaveException extends RuntimeException{
    public BookSaveException (String message, IOException cause){
        super(message, cause);
    }
}

Block of code changed to:

    @Override
    public void saveBookInfo(Book book) {
        try (PrintWriter writer = new PrintWriter(new FileWriter(filePath, true))) {
            String formattedEntry = String.format("\"%s\" %s - %s", book.getTitle(), book.getAuthor(), book.getBookStatus());
            writer.println(formattedEntry);
        } catch (IOException exception) {
            throw new BookSaveException ("An error occurred while saving the book information.", exception);
        }
    }